### PR TITLE
Feature/operations

### PR DESCRIPTION
**Pull request by knightstick**
Implements basic arithmetic (+, -, *, /) for the Minions.

For a while I had a special bonus extra case, where, if you divide by zero, the state is set as "banana", but, my fancy pants refactoring broke it, so, I took it out again.

### CODE REVIEW

**Comment by DuBistKomisch** [comment]
🍌

**Comment by mfeckie** [comment]
I'm very sad that status was not set to banana... like VERY sad.  In fact now I know that it was there, I demand that it be there or I will 😢

**Comment by knightstick** [comment]
@mfeckie If you can come up with a nice `&()` style anonymous function that returns "banana" when you try to divide by 0, then I'm happy to update.

File: test/lib/minion_test.exs [hunk 1]
```diff
@@ -39,5 +39,52 @@ defmodule AssembleTheMinions.MinionTest do
...
+
+    tim_count = AssembleTheMinions.Minion.current_count(:tim)
+
+    assert tim_count == 6
+  end
+
+  test "Minions can perform division" do
+    AssembleTheMinions.Minion.start_link(:div)
+
+    # Is there an equivalent to Ruby's 4.times { }?
```
Comment (mfeckie) [inline test/lib/minion_test.exs:78]:
``` elixir
for _ <- 1..4 do
  AssembleTheMinions.Minion.count(:div)
end
```

would work

**Comment by mfeckie** [comment]
@knightstick Challenge accepted https://github.com/ElixirStudyGroup/assemble_the_minions/commit/4fc026ab1b8377d8673deaba1e352b5d57b5a72b

**Pull request closed without merging**